store/db.go

```diff
@@ -91,7 +92,7 @@ func (db *DB) Open() error {
 	if err != nil {
 		unlockErr := db.dl.unlock()
 		if unlockErr != nil {
-			err = fmt.Errorf("[%s, %s]", err, unlockErr)
+			err = errwrap.Wrap(unlockErr, err)
```

Comment:
Not sure if this is exactly right - unlocking the db didn't happen because of an error in `err` variable. It is a parallel error, but I guess that errwrap doesn't have a tree of errors, only a stack.